packages/app/src/features/my-portfolio/components/borrow-table/components/EModeIndicator.tsx

```diff
@@ -1,21 +1,22 @@
 import { EModeCategoryId } from '@/domain/e-mode/types'
-import { Info } from '@/ui/molecules/info/Info'
-
-import { EModeButton } from './EModeButton'
+import { Button } from '@/ui/atoms/new/button/Button'
+import { EModeBadge } from '@/ui/molecules/e-mode-badge/EModeBadge'
+import { testIds } from '@/ui/utils/testIds'
+import { useBreakpoint } from '@/ui/utils/useBreakpoint'
 
 interface EModeIndicatorProps {
   eModeCategoryId: EModeCategoryId
   onButtonClick: () => void
 }
 
 export function EModeIndicator({ eModeCategoryId, onButtonClick }: EModeIndicatorProps) {
+  const isNotMobile = useBreakpoint('sm')
   return (
-    <div className="mt-1.5 ml-4 flex items-center gap-2" onClick={(e) => e.stopPropagation()}>
-      <div className="flex flex-row items-center gap-0.5">
-        <div className="typography-label-5 text-secondary">E-Mode</div>
-        <Info>Efficiency mode (E-Mode) increases your LTV for a selected category of assets up to 97%.</Info>
-      </div>
-      <EModeButton categoryId={eModeCategoryId} onClick={onButtonClick} />
+    <div className="flex w-full items-center justify-between gap-2">
+      <EModeBadge categoryId={eModeCategoryId} />
+      <Button size="s" variant="tertiary" onClick={onButtonClick} data-testid={testIds.component.EModeButton}>
+        {isNotMobile && 'Change'} E-Mode
```

Comment:
Maybe the same can be achieved with
```suggestion
        <span className='hidden sm:visible'>Change</span> E-Mode
```
?

Reply:
Worst case I guess we can do
```
 <span className='hidden sm:visible'>Change E-Mode</span>
  <span className='sm:hidden'> E-Mode</span>
```